### PR TITLE
Chore: Allow load metrics emission without file info

### PR DESCRIPTION
**Pull request by JustinHoldstock**
Allows for file info to be request external to preview, and still report timings. 

*Note: Updates are required in Box UI Elements before this can be merged, otherwise false reports will be generated.*

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @JustinHoldstock has signed the CLA. Thanks for the pull request!

File: src/lib/events.js [hunk 1]
```diff
@@ -44,7 +44,6 @@ export const ERROR_CODE = {
     FLASH_NOT_ENABLED: 'error_flash_not_enabled'
 };
 
-export const PREVIEW_LOAD_EVENT = '';
```
Comment (jeremypress) [inline src/lib/events.js]: Why are we removing this?

Comment (JustinHoldstock) [reply src/lib/events.js]: It's not used anywhere AND it's an empty string

Comment (JustinHoldstock) [reply src/lib/events.js]: We now use the below `LOAD_METRIC.previewLoadEvent`

**Pull request merged**